### PR TITLE
KAZOO-5744: fixes for webhook filtering and other purposes

### PR DESCRIPTION
**Pull request by jamesaimonetti**
Fix setting outbound call id for conference dial

Fix application vars showing up on bridge strings

### CODE REVIEW

File: applications/ecallmgr/src/ecallmgr.hrl [hunk 1]
```diff
@@ -256,9 +256,9 @@
                               ,{<<"Media-Encryption-Enforce-Security">>,<<"sdp_secure_savp_only">>}
                               ,{<<"Media-Files-Separator">>, <<"playback_delimiter">>}
                               ,{<<"Media-Webrtc">>, <<"media_webrtc">>}
-                              ,{<<"Origination-Call-ID">>, <<"sip_origination_call_id">>}
+                              ,{<<"Origination-Call-ID">>, <<"origination_uuid">>}
```
Comment (lazedo) [inline applications/ecallmgr/src/ecallmgr.hrl]: why this change ?

Comment (jamesaimonetti) [reply applications/ecallmgr/src/ecallmgr.hrl]: @lazedo I can't find any mention of `sip_origination_call_id` in FreeSWITCH wiki nor did I see the asked-for call-id being set, but setting `origination_uuid` worked and is found in [Channel Variables](https://freeswitch.org/confluence/display/FREESWITCH/Channel+Variables)

Comment (lazedo) [reply applications/ecallmgr/src/ecallmgr.hrl]: @jamesaimonetti its set by us so that we can bind to a know call-id from the originating process (fax worker uses it) for calls that have 2+ destinations (2+ /= uuids) and the first one fails we can still get events from the other legs.
ecallmgr_call_event sets `{<<"Origination-Call-ID">>, kzd_freeswitch:origination_call_id(Props)}` from that value and `kapi_call:publish_event` prefers it when setting the routing key.
i propose we leave `Origination-Call-ID` as-is and change fax_worker to send that instead of `Outbound-Call-ID` and have stepswitch forward `Origination-Call-ID` and  `Outbound-Call-ID` (with value = `Origination-Call-ID).

this can also be useful to track quickcall

File: applications/stepswitch/src/stepswitch_originate.erl [hunk 2]
```diff
@@ -271,34 +271,35 @@ build_originate(#state{endpoints=Endpoints
...
       ,{<<"Ignore-Early-Media">>, kz_json:get_value(<<"Ignore-Early-Media">>, OffnetReq)}
+      ,{<<"Loopback-Bowout">>, kz_json:get_value(<<"Loopback-Bowout">>, OffnetReq)}
       ,{<<"Media">>, kz_json:get_value(<<"Media">>, OffnetReq)}
-      ,{<<"Hold-Media">>, kz_json:get_value(<<"Hold-Media">>, OffnetReq)}
-      ,{<<"Presence-ID">>, kz_json:get_value(<<"Presence-ID">>, OffnetReq)}
-      ,{<<"Outbound-Callee-ID-Number">>, kz_json:get_value(<<"Outbound-Callee-ID-Number">>, OffnetReq)}
+      ,{<<"Msg-ID">>, MsgId}
+      ,{<<"Originate-Immediate">>, kz_json:get_value(<<"Originate-Immediate">>, OffnetReq)}
+      ,{<<"Origination-Call-ID">>, kz_json:get_value(<<"Origination-Call-ID">>, OffnetReq)}
+      ,{<<"Outbound-Call-ID">>, kz_json:get_value(<<"Outbound-Call-ID">>, OffnetReq)}
```
Comment (lazedo) [inline applications/stepswitch/src/stepswitch_originate.erl]: can we default `Outbound-Call-ID` to `Origination-Call-ID` ? we still want the originate to use the provided call_id. we can also do this in ecallmgr_originate.
